save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/docker/DockerAgentRunner.kt

```diff
@@ -254,4 +256,4 @@ class DockerAgentRunner(
 /**
  * @param id
  */
-private fun containerName(id: String) = "save-execution-$id"
+private fun containerName(id: String) = "save-execution-$id-${abs(Random.nextInt())}"
```

Comment:
Maybe format it to a fixed number of digits to make container names look more consistent with each other?

Reply:
Add interval of possible values for random